dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoDualWriterClient.java

```diff
@@ -174,74 +174,106 @@ public void setRange(int range) {
 
     @Override
     public Long append(final String key, final String value) {
+        return this.d_append(key, value).getResult();
+    }
+
+    @Override
+    public OperationResult<Long> d_append(final String key, final String value) {
```

Comment:
was implementing the `d_xxx` needed for the dual write fix, or they were done as part of our conversation?